lib/ovirt/template.rb

```diff
@@ -152,9 +152,7 @@ def build_clone_xml(options)
             end
           end
 
-          xml.os(:type => options[:os_type] || os_type) do
-            xml.boot(:dev => 'hd')
-          end
+          xml.os(:type => options[:os_type] || os_type)
```

Comment:
When running a PXE or ISO provision we use the skeletal clone option which uses the blank template as the source to avoid copying gigabytes of data that will be deleted.  https://github.com/ManageIQ/ovirt/blob/master/lib/ovirt/template.rb#L109